diff --git a/node_modules/react-native-sound/android/src/main/java/com/zmxv/RNSound/Sound.kt b/node_modules/react-native-sound/android/src/main/java/com/zmxv/RNSound/Sound.kt
index 749ea14..3316eab 100644
--- a/node_modules/react-native-sound/android/src/main/java/com/zmxv/RNSound/Sound.kt
+++ b/node_modules/react-native-sound/android/src/main/java/com/zmxv/RNSound/Sound.kt
@@ -9,7 +9,7 @@ import android.media.MediaPlayer.OnPreparedListener
 
 import android.util.Log
 import com.facebook.react.bridge.Arguments
-import com.facebook.react.bridge.BridgeReactContext.RCTDeviceEventEmitter
+import com.facebook.react.modules.core.DeviceEventManagerModule.RCTDeviceEventEmitter
 import com.facebook.react.bridge.Callback
 import com.facebook.react.bridge.ReactApplicationContext
 
